README.md

```diff
@@ -20,6 +56,14 @@ git clone git@github.com:awslabs/aws-checksums.git
 cmake -DCMAKE_PREFIX_PATH=<install-path> -DCMAKE_INSTALL_PREFIX=<install-path> -S aws-checksums -B aws-checksums/build
 cmake --build aws-checksums/build --target install
 
+git clone git@github.com:awslabs/aws-c-cal.git
```

Comment:
Isn't awslabs an internal repository? Shouldn't it point to aws one?

Reply:
"awslabs" is public, and as far as I know, separate from "aws".

Reply:
"awslabs" is for stuff that's still a little experimental. But most of its repos are public

Reply:
sometimes a repo will "graduate" from "awslabs" to "aws"